rebar.config

```diff
@@ -4,7 +4,7 @@
 {deps, [
 	{setup, "2.1.0"},
 	{jobs, "0.9.0"},
-	{prometheus, {git, "https://github.com/deadtrickster/prometheus.erl.git", {branch, "master"}}},
+	{prometheus, {git, "https://github.com/deadtrickster/prometheus.erl.git", {branch, "v4.8.1"}}},
```

Comment:
can we make that 4.8.2 ?

Reply:
Also it make sense to update prometheus in eradius.

Reply:
@vkatsuba you mean the `master` branch is also used in eradius?

Reply:
Most commits in 4.8.2 are quite old. Going from master to 4.8.1 would actually mean we are suddenly using versions that are older that what we have been using in the last months and have tested in QA.

Reply:
Hum, totally true. I'll use 4.8.2 then.

Reply:
> @vkatsuba you mean the `master` branch is also used in eradius?

https://github.com/travelping/eradius/blob/master/applications/eradius_prometheus_collector/rebar.config#L2 here where is need update I suppose if we use it in our repos. Provide release of eradius and include it into ergw_aaa.